pyoptsparse/pySNOPT/source/f2py/snopt.pyf

```diff
@@ -424,14 +424,14 @@ python module snopt ! in
             integer :: ns
             integer :: ninf
             double precision :: sinf
-            character*8 intent(in,out), dimension(lencu) :: cu
             double precision intent(out) :: obj
+            character*8 intent(inplace), dimension(lencu) :: cu
             integer optional,check(len(cu)==lencu),depend(cu) :: lencu=len(cu)
             integer dimension(leniu) :: iu
             integer optional,check(len(iu)>=leniu),depend(iu) :: leniu=len(iu)
             double precision dimension(lenru) :: ru
             integer optional,check(len(ru)>=lenru),depend(ru) :: lenru=len(ru)
-            character*8 intent(in,out), dimension(lencw) :: cw
+            character*8 intent(inplace), dimension(lencw) :: cw
```

Comment:
also, this trick? has anything to do with making the array fortran-contiguous or is it just related to the type change above?

Reply:
This is what the "official" pyf file looks like, so I just took it here. I didn't want it to be `in, out` since that would return those and we have no use for these variables.

Reply:
And somehow `inout` didn't work but this did.